app/templates/dashboard_all.html

```diff
@@ -32,6 +32,9 @@ <h1 class="govuk-heading-xl">{{ pageHeading }}</h1>
                 <h2 class="govuk-accordion__section-heading">
                     <span class="govuk-accordion__section-button" id="funds-heading-{{ fund['fund_data']['short_name'] }}">
                         {{ fund["fund_data"]["name"] }}
+                        {% if fund["fund_data"]["funding_type"] == "EOI" %}
+                            ({% trans %}Expression of interest{% endtrans %})
```

Comment:
Don't think we want brackets, we want it to be seamless from what's there now. So like
```{% if EOI %}
    &nbsp;-&nbsp;Expression of Interest
{% endif %}
```

Reply:
https://github.com/communitiesuk/funding-service-design-frontend/pull/640/commits/2d512651dfc76c39d79b165f16789b9d17929902